tests/affiliatedcertification/tests/affiliated_certification_invalid_operator.go

```diff
@@ -138,22 +140,24 @@ var _ = Describe("Affiliated-certification invalid operator certification,", fun
 			Expect(err).ToNot(HaveOccurred(), "Error deploying operator "+
 				affiliatedcertparameters.UncertifiedOperatorPrefixSriov)
 
-			var installPlan *v1alpha1.InstallPlan
 			Eventually(func() bool {
-				installPlan, err = affiliatedcerthelper.GetInstallPlanByCSV(affiliatedcertparameters.TestCertificationNameSpace,
+				installPlan, err := affiliatedcerthelper.GetInstallPlanByCSV(affiliatedcertparameters.TestCertificationNameSpace,
 					affiliatedcertparameters.UncertifiedOperatorFullSriov)
 				if err == nil {
-					return installPlan.Status.Phase != "" &&
-						installPlan.Status.Phase != v1alpha1.InstallPlanPhasePlanning
+					if installPlan.Status.Phase != "" &&
```

Comment:
@theemj perhaps we can drop the return true/ false, and just return installpan.status.phase != " "... since this is already a boolean, WDYT?